### ISSUE TITLE
Query Admin - Passthrough query no longer works

### ISSUE THREAD

**Issue by gsteimer** [tsgrp/HPI#1700]
The passthrough query tool in the admin is no longer functioning properly.  Even a simple query like:

```sql
SELECT COUNT(n.id) FROM alf_node n
```

Results in an error on the OC side:

```
2018-04-27 12:59:39,794	ajp-apr-8009-exec-3	ERROR	AbstractAlfrescoEmbQueryImpl	SQL Excepton thrown running read-only query: SELECT COUNT(n.id) FROM alf_node n
org.postgresql.util.PSQLException: Cannot change transaction read-only property in the middle of a transaction.
	at org.postgresql.jdbc2.AbstractJdbc2Connection.setReadOnly(AbstractJdbc2Connection.java:718) ~[postgresql-9.4-1201-jdbc41.jar:9.4]
	at org.apache.commons.dbcp.DelegatingConnection.setReadOnly(DelegatingConnection.java:377) ~[commons-dbcp-1.4-DBCP330.jar:1.4]
	at org.apache.commons.dbcp.DelegatingConnection.setReadOnly(DelegatingConnection.java:377) ~[commons-dbcp-1.4-DBCP330.jar:1.4]
	at org.apache.commons.dbcp.PoolingDataSource$PoolGuardConnectionWrapper.setReadOnly(PoolingDataSource.java:338) ~[commons-dbcp-1.4-DBCP330.jar:1.4]
	at com.tsgrp.opencontent.alfresco.query.OCSQLDbQueryManager.executeQuery(OCSQLDbQueryManager.java:102) ~[oc-hpiDemoAlf.2.alfrescoEmb-2.6.jar:2.6]
	at com.tsgrp.opencontent.alfresco.query.OCSQLDbQueryManager.executeReadQuery(OCSQLDbQueryManager.java:57) ~[oc-hpiDemoAlf.2.alfrescoEmb-2.6.jar:2.6]
	at com.tsgrp.opencontent.alfresco.query.AbstractAlfrescoEmbQueryImpl.enhancedPassthroughReadQuery(AbstractAlfrescoEmbQueryImpl.java:2018) [oc-hpiDemoAlf.2.alfrescoEmb-2.6.jar:2.6]
...
com.tsgrp.opencontent.core.exception.OCRuntimeException: Failed writing file
	at com.tsgrp.opencontent.core.action.executer.ExecuteReadOnlyQueryActionExecuter.executeImpl(ExecuteReadOnlyQueryActionExecuter.java:87) ~[oc-hpiDemoAlf.1.core-2.6.jar:2.6]
	at com.tsgrp.opencontent.core.action.executer.AbstractActionExecuter.execute(AbstractActionExecuter.java:196) ~[oc-hpiDemoAlf.1.core-2.6.jar:2.6]
	at com.tsgrp.opencontent.core.action.executer.AbstractActionExecuter.evaluateAndExecute(AbstractActionExecuter.java:135) ~[oc-hpiDemoAlf.1.core-2.6.jar:2.6]
	at com.tsgrp.opencontent.core.action.service.CoreActionService.execute(CoreActionService.java:474) ~[oc-hpiDemoAlf.1.core-2.6.jar:2.6]
```

**Comment by gsteimer** [comment]
Per @jharkins2 - this may only affect Alfresco's that use Postgres as the underlying database.

**Comment by jharkins2** [comment]
Right now in OCSQLDbQueryManager, line 102, we are doing:
```
conn.setReadOnly(true);
```
The Javadoc for setReadOnly mentions:
```
Throws:
SQLException - if a database access error occurs, this method is called on a closed connection or this method is called during a transaction
```

and we are running this inside of a transaction, since the `CoreContext.getQuery().enhancedPassthroughReadQuery` we run everything in opens a transaction.

It looks like not all drivers actually do this, since we can get away with it in MySQL.
We can either no longer run this in a transaction or don't do the setReadOnly.

**Comment by CRdouglas** [comment]
QUERY ADMIN: Removing line to help fix issue #1700. Reasoning: You can't setReadOnly inside of a JDBC Transaction. Since this doesn't actually block non-readonly queries, only "provides hints to the driver", we can get away with just removing it.

Link: https://github.com/tsgrp/HPI/issues/1700

CR - James H/George S
Rev 18887
